gen.py

```diff
@@ -26,15 +35,28 @@ def get_github_url(repo, path=''):
     return 'https://github.com/IATI/{0}/blob/{1}/{2}'.format(repo, github_branches[repo], path)
 
 def human_list(l):
-    """
-    Returns a human friendly version of a list. Currently seperates list items
-    with comas, but could be extended to insert 'and'/'or' correctly.
+    """Return a human-friendly version of a list.
+    Currently seperates list items with commas, but could be extended to insert 'and'/'or' correctly.
+
+    Args:
+        l (list): The list to be made human-friendly.
 
+    Returns:
+        str: The human-friendly represention of the list.
     """
     return ', '.join(l)
 
-
 def lookup_see_also(standard, mapping, path):
+    """Return a generator object containing ???
+
+    Args:
+        standard (str): Unknown
+        mapping (list): Unknown
+        path (str): Unknown
+
+    Returns:
+        generator or str: Unknown
```

Comment:
generator (str): Generates a series of unicode strings for Activity Elements in Schema (I think).